src/App.js

```diff
@@ -1,64 +1,63 @@
-import React, { Component } from 'react';
+/*
+   Edited by: Maria Babko
+   Date March 2021
+
+   Renders a Todolist page.
+*/
+import React, { useState } from 'react';
+
+import Alert from 'react-bootstrap/Alert';
+import ManageTodolist from './components/ManageTodolist';
 
 /**
- * App class runs entire application
- */
-class App extends Component {
+  * Sends POST request to TodolistAPI to add todo-list item.
```

Comment:
-1 Incorrect Comment, the App function specifically doesnt do any POST requests.
You had the correct comment up above as a file header comment which is a good.

It is important to not have any comments that misdirect or confuse the developer reading your code.